app/models/season/phase_switcher.rb

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
   def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    if phase.in?(whitelisted_phases)
```

Comment:
Ok, get it. However, it's a dev-env-only feature anyways, so if someone is trying to shoot down their server with invalid params... their problem. To me "silent failing" sounds like hiding potential problems and increasing technical dept. Therefore I'd vote for using the error approach, it's simply quicker feedback that there's something wrong.